openvdb/openvdb/python/pyproject.toml

```diff
@@ -0,0 +1,36 @@
+[build-system]
+requires = [
+    "scikit_build_core",
+    "pybind11",
```

Comment:
This could probably be pinned to avoid any future surprises.

Reply:
Sure, that makes sense. Pinning to 2.10.3 unless others feel differently.